templates/hugo/files/.gitmodules

```diff
@@ -0,0 +1,4 @@
+
+[submodule "themes/minimal"]
```

Comment:
No submodules.  We do not give customers submodules.  They're a time bomb waiting to go off, usually sooner rather than later.